cli/README.md

```diff
@@ -13,15 +13,24 @@ Then, to build the open-api client run the following in the open-api folder:
 
     $ ./bin/generate-open-api.sh
 
-To run the Immich CLI from source, run the following in the cli folder:
```

Comment:
I don't think we should only provide instructions on how to run inside vscode. Lets leave the instructions on how to run as they were.